### ISSUE TITLE
amp-date-picker: allow to hide keyboard shortcuts panel

### ISSUE THREAD

**Issue by rpvsilva** [ampproject/amphtml#25608]
## Describe the new feature or change to an existing feature you'd like to see

As in react-date, by AirBnB, we can set a flag to hide the keyboard shortcuts panel, in amp we should be able to set a flag to do this too

In react-date the flag is `hideKeyboardShortcutsPanel`

**Comment by nainar** [comment]
Hi @rpsilva would you be interested in making that change?

**Comment by rpvsilva** [comment]
Hi @nainar yes I'm interested in making this change, could you help how to configure the project in my local machine? Or is there a guide to get started?

**Comment by rpvsilva** [comment]
> Hi @nainar yes I'm interested in making this change, could you help how to configure the project in my local machine? Or is there a guide to get started?

@nainar I have finally configured the project and I have done the issue. I have committed to my branch. What's the next step?
